canonical-websites/maas.io
# Migrate /install to VBT

## Done

Migrate /install to VBT

## QA

- ./run then go to http://0.0.0.0:8006/
- check in s/m and l
- Copy to clipboard still WIP

## Issue / Card

Fixes #147 

## Screenshots

These issues will be fixed once the latest VBT lands

![screen shot 2017-07-05 at 09 22 01](https://user-images.githubusercontent.com/2152/27856089-d02f682a-6164-11e7-8cf4-cf86ce0ca8ef.png)
